.github/workflows/ci.yml

```diff
@@ -15,30 +15,46 @@ defaults:
 
 jobs:
   build:
+    name: ${{ matrix.name }}
     runs-on: ubuntu-18.04
     strategy:
       fail-fast: false
       matrix:
         include:
+
           # Oldest supported versions
-          - enable_cuda: true
+          - name: Linux (CUDA 10.2, Python 3.7, PyTorch 1.11)
+            enable_cuda: true
             cuda: "10.2.89"
-            gcc: "8.5.0"
+            gcc: "8.5.*"
             nvcc: "10.2"
-            python: "3.7"
-            pytorch: "1.11.0"
-          # Latest supported versions
-          - enable_cuda: true
+            python: "3.7.*"
+            pytorch: "1.11.*"
+
+          # Older supported versions
+          - name: Linux (CUDA 11.2, Python 3.9, PyTorch 1.11)
+            enable_cuda: true
+            cuda: "11.2.2"
+            gcc: "10.3.*"
+            nvcc: "11.2"
+            python: "3.9.*"
+            pytorch: "1.11.*"
+
+          # Latest supported versions (with CUDA)
+          - name: Linux (CUDA 11.2, Python 3.10, PyTorch 1.12)
```

Comment:
All of these are older versions.  How about CUDA 11.8, Python 3.11, and PyTorch 1.13?

Reply:
Python 3.11 and PyTorch 1.13 are not available because there are no corresponding TorchANI packages.

Reply:
Changed to CUDA 11.7. CUDA 11.8 isn't available because of the `cuda-toolkit` action not yet supporting it.